Packs/Illumio/Integrations/IllumioCore/IllumioCore_test.py

```diff
@@ -57,222 +56,277 @@ def test_test_module(requests_mock, mock_client):
+                ),
+                InvalidValueError,
+        ),
+        (
+                {
+                    "port": 8443,
+                    "protocol": "dummy",
+                    "start_time": "2022-07-17T12:58:33.528Z",
+                    "end_time": "2022-07-18T12:58:33.529Z",
+                    "policy_decisions": ["potentially_blocked"],
+                },
+                "{} is an invalid value for {}. Possible values are: {}.".format("dummy", "protocol", VALID_PROTOCOLS),
+                InvalidValueError,
+        ),
+    ],
+)
+def test_traffic_analysis_command_for_invalid_arguments(args, err_msg, err_type, mock_client):
     """
-    Test case scenario for execution of illumio-traffic-analysis-command when invalid argument provided.
+    Test case scenario for execution of traffic-analysis-command when invalid argument provided.
 
     Given:
-        - command arguments for illumio_traffic_analysis_command
+        - command arguments for traffic_analysis_command
     When:
-        - Calling `illumio_traffic_analysis_command` function
+        - Calling `traffic_analysis_command` function
     Then:
         - Returns a valid error message
     """
-    from IllumioCore import illumio_traffic_analysis_command
-    with pytest.raises((ValueError, IllumioException, InvalidPortException, InvalidSingleSelectException,
-                        InvalidMultiSelectException)) as err:
-        illumio_traffic_analysis_command(mock_client, args)
-    assert str(err.value) == err_msg
+    from IllumioCore import traffic_analysis_command
+
+    with pytest.raises(err_type) as err:
+        traffic_analysis_command(mock_client, args)
+        assert str(err.value) == err_msg
 
 
-def test_illumio_traffic_analysis_success(requests_mock, mock_client):
+def test_traffic_analysis_success(requests_mock, mock_client):
     """
-    Test case scenario for successful execution of illumio_traffic_analysis_command function.
+    Test case scenario for successful execution of traffic-analysis-command function.
 
     Given:
-        - command arguments for illumio_traffic_analysis
+        - command arguments for traffic_analysis
     When:
-        - Calling `illumio_traffic_analysis_Command` function
+        - Calling `traffic_analysis_Command` function
     Then:
-        - Returns a valid output
+        - Returns a valid raw_response.
     """
-    from IllumioCore import illumio_traffic_analysis_command
+    args = {
+        "port": 8443,
+        "protocol": "tcp",
+        "policy_decisions": "potentially_blocked",
+        "start_time": "2022-07-17T12:58:33.528Z",
+        "end_time": "2022-07-18T12:58:33.529Z",
+    }
 
-    args = {'port': 8443, 'protocol': "tcp", 'policy_decisions': 'potentially_blocked',
-            'start_time': "2022-07-17T12:58:33.528Z", 'end_time': "2022-07-18T12:58:33.529Z"}
+    json_data = util_load_json(os.path.join(DIRECTORY_PATH, "test_data/traffic_analysis_success_response.json"))
```

Comment:
Change `DIRECTORY_PATH` to include the `test_data` directory. 

Also note that having the `/` literal makes it pointless to use os.path.join because it won't work cross-platform - the correct call would be `os.path.join(PATH, "data_dir", "file.json")`

Reply:
Okay, I'll make the change